src/Microsoft.AspNetCore.Mvc.Core/Internal/MvcCoreLoggerExtensions.cs

```diff
@@ -636,6 +637,11 @@ static MvcCoreLoggerExtensions()
                LogLevel.Debug,
                46,
                "Could not find a value in the request with name '{ModelName}' of type '{ModelType}'.");
+
+            _modelBindingOfPropertyShortCircuit = LoggerMessage.Define<Type, string>(
+               LogLevel.Debug,
+               47,
+               "Could not bind to property '{PropertyContainerType}.{PropertyName}' as it's binding information rejected it.");
```

Comment:
"disallowed it for the current request" instead of "rejected it"? The predicate is meant to be per-request choice as opposed to `BindNever` which always rejects it.

Reply:
This could be an action argument. Maybe

> Skipped binding parameter '{ParameterName}' since it's binding information disallowed it for the current request.